LibGAP.jl/src/julia_to_gap.jl

```diff
@@ -23,8 +23,17 @@ julia_to_gap(x::BigInt)  = MakeObjInt(x)
 
 ## Rationals
 function julia_to_gap(x::Rational{T}) where T <: Integer
-    numer = julia_to_gap(numerator(x))
-    denom = julia_to_gap(denominator(x))
+    denom_julia = denominator(x)
+    numer_julia = numerator(x)
+    if denom_julia == 0
+        if numer_julia >= 0
+            return GAP.Globals.infinity
+        else
+            return -GAP.Globals.infinity
```

Comment:
Perhaps add `Resolves #162` to the relevant commit message?

Reply:
I thought about it, and I would rather make a comment here in the PR, because I think we wanted to keep the commit messages clear from the `Closes #whatever` messages?

Reply:
I certainly was never part of this "we" you mention ;-). But I really don't mind either way in this case, as long as the issues get referenced and closed

Reply:
Maybe it was not "we", but "me", I am not sure anymore ;)